HW_4_Task_2/HW_4_Task_2.Tests/ListTest.cs

```diff
@@ -0,0 +1,84 @@
+﻿using NUnit.Framework;
+using System;
+
+namespace HW_4_Task_2.Tests
+{
+    [TestFixture]
+    class ListTest
+    {
+        private List list = new List();
```

Comment:
```suggestion
        private List list = new();
```
Начиная с C# 9 вроде уже ни в какой ситуации не надо писать тип дважды